git_machete/cli.py

```diff
@@ -99,6 +99,153 @@ def launch(orig_args: List[str]) -> None:
+
+        arg_parser = argparse.ArgumentParser(
+            prog='git machete', usage=short_usage(), add_help=False,
+            argument_default=argparse.SUPPRESS)
+
+        subparsers = arg_parser.add_subparsers(dest='command', required=True)
+
+        add_parser = subparsers.add_parser('add', argument_default=argparse.SUPPRESS, help=None)
+        # TODO help
+        add_parser.add_argument('branch')
+        add_parser.add_argument('-o', '--onto')
+        add_parser.add_argument('-R', '--as-root', action='store_true')
+        add_parser.add_argument('-y', '--yes', action='store_true')
+
+        advance_parser = subparsers.add_parser('advance')
+        advance_parser.add_argument('-y', '--yes', action='store_true', default=argparse.SUPPRESS)
+
+        anno_parser = subparsers.add_parser('anno')
+        # possible values of 'annotation_text' include: [], [''], ['some_val'], ['text_1', 'text_2']
+        anno_parser.add_argument('annotation_text', nargs='*')
+        anno_parser.add_argument('-b', '--branch', default=argparse.SUPPRESS)
+        anno_parser.add_argument(
+            '-H', '--sync-github-prs', action='store_true', default=argparse.SUPPRESS)
+
+        delete_unmanaged_parser = subparsers.add_parser('delete-unmanaged')
+        delete_unmanaged_parser.add_argument(
+            '-y', '--yes', action='store_true', default=argparse.SUPPRESS)
+
+        diff_full_parser = subparsers.add_parser('diff', aliases=['d'])
+        diff_full_parser.add_argument('branch')
+        diff_full_parser.add_argument(
+            '-s', '--stat', action='store_true', default=argparse.SUPPRESS)
+
+        discover_parser = subparsers.add_parser('discover', argument_default=argparse.SUPPRESS)
+        discover_parser.add_argument('-C', '--check-out-since')
+        discover_parser.add_argument('-l', '--list-commits', action='store_true')
+        discover_parser.add_argument('-r', '--roots')
+        discover_parser.add_argument('-y', '--yes', action='store_true')
+
+        edit_parser = subparsers.add_parser('edit', aliases=['e'])
+
+        file_parser = subparsers.add_parser('file')
+
+        fork_point_parser = subparsers.add_parser('fork-point', argument_default=argparse.SUPPRESS)
+        fork_point_parser.add_argument('branch', default=None)
+        fork_point_parser.add_argument('--inferred', action='store_true')
+        fork_point_parser.add_argument('--override-to')
+        fork_point_parser.add_argument('--override-to-inferred', action='store_true')
+        fork_point_parser.add_argument('--override-to-parent', action='store_true')
+        fork_point_parser.add_argument('--unset-override', action='store_true')
+
+        format_parser = subparsers.add_parser('format')
+
+        github_parser = subparsers.add_parser('github')
+        github_parser.add_argument('subcommand', choices=['anno-prs', 'create-pr', 'retarget-pr'])
+        github_parser.add_argument('--draft', action='store_true', default=argparse.SUPPRESS)
```

Comment:
I wonder how it should look like with `github checkout-pr N`. Probably another subparser has to be added, sth like:
```suggestion
       github_parser.add_argument('--draft', action='store_true', default=argparse.SUPPRESS)
       github_parser.add_argument('integer', metavar='N', type=int, help='an integer for the PR number')
```

Reply:
Another argument, not a subparser, but yes, overall github subcommand choices would have to be expanded and argument 'N' would have to be added.